web3/_utils/module_testing/personal_module.py

```diff
@@ -343,3 +346,61 @@ def test_personal_sign_typed_data_deprecated(
             )
             assert signature == expected_signature
             assert len(signature) == 32 + 32 + 1
+
+
+class GoEthereumAsyncPersonalModuleTest:
+
+    @pytest.mark.asyncio
+    async def test_async_sign_and_ec_recover(self,
```

Comment:
all tests should start with this same `test_async_...`

Reply:
fixed